styles.css

```diff
@@ -0,0 +1,214 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #222;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  min-height: 100vh;
+
+  color: white;
+
+  font-family: "Poppins", sans-serif;
+}
+
+div.container {
+  position: relative;
+}
+
+div.card {
+  position: relative;
+  height: 490px;
+  width: 350px;
+
+  overflow: hidden;
+
+  border: 3px solid white;
+  border-radius: 30px;
+
+  background-color: #131313;
+
+  box-shadow: 10px 10px 40px rgba(0, 0, 0, 0.9);
+}
+
+div.imgbox {
+  width: 350px;
+
+  padding-top: 5em;
+  /* border: 1px solid white; */
+
+  position: absolute;
+
+  z-index: 100;
+
+  /* overflow: hidden; */
+}
+
+.imgbox img {
+  width: 100%;
+  /* z-index: 100; */
+}
+
+div.card::before {
+  content: "";
+
+  position: absolute;
+  top: 0;
+
+  clip-path: circle(180px at 90% 15%);
+  height: 100%;
+  width: 100%;
+
+  background-color: #ec060e;
+
+  /* z-index: ; */
+}
+
+div.card:hover::before {
+  clip-path: circle(300px at 80% -15%);
+  background-color: #b5070d;
+  transition: 0.5s ease-in-out;
+}
+
+div.card:hover img {
+  transform: translate(-5%, -25%) rotate(-38deg);
+
+  transition: 0.5s;
+}
+
+div.card::after {
+  content: "Nike";
+
+  position: absolute;
+  left: -40px;
+  top: 150px;
+
+  font-size: 12em;
+  font-weight: 600;
+  font-style: italic;
+
+  color: #222;
+
+  /* z-index: ; */
+}
+
+div.contentbox {
+  width: 100%;
+  position: absolute;
+
+  bottom: 15px;
+  height: 100px;
+  margin: 0.5em 0;
+  text-align: center;
+
+  letter-spacing: 1px;
+
+  z-index: 5;
+}
+
+div.size {
+  opacity: 0;
+  visibility: hidden;
+  display: flex;
+  justify-content: center;
+
+  margin: 0.5em 0;
+}
+
+.size span {
+  border: 1px solid;
+  margin: 2px 5px;
+  padding: 1px 4px;
+
+  border-radius: 4px;
+}
+
+div.color {
+  opacity: 0;
+  visibility: hidden;
+  display: flex;
+
+  justify-content: center;
+
+  margin-bottom: 1em;
+}
+
+.color span {
+  height: 15px;
+  width: 10px;
+
+  margin: 7px 5px;
+  padding: 1px 4px;
+
+  border-radius: 50%;
+}
+
+.color span:nth-child(2) {
+  background-color: #ec060e;
+}
+.color span:nth-child(3) {
+  background-color: blue;
+}
+.color span:nth-child(4) {
+  background-color: green;
+}
+.color span:nth-child(5) {
+  background-color: yellowgreen;
+}
+
+.size span:hover {
+  color: black;
+  background: #fff;
+
+  cursor: pointer;
+}
+
+.contentbox a {
+  opacity: 0;
+  color: black;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  font-weight: 600;
+  letter-spacing: 0px;
```

Comment:
what is default value of letter-spacing ?

Reply:
Initially I kept it to 1px, but then removed it as it wasn't looking good. Default Value is 0px only.